packages/instantsearch.js/src/connectors/frequently-bought-together/connectFrequentlyBoughtTogether.ts

```diff
@@ -31,7 +29,7 @@ export type FrequentlyBoughtTogetherRenderState<
   /**
    * The matched recommendations from Algolia API.
    */
-  items: Array<Hit<THit>>;
+  items: Array<AlgoliaHit<THit>>;
```

Comment:
all recommend changes Hit -> AlgoliaHit is because __position and __queryId aren't actually present in these connectors